filebeat/prospector/prospector_log.go

```diff
@@ -112,17 +114,69 @@ func (l *Log) Run() {
 	}
 }
 
+// globPattern detects the use of "**" and expands it to standard glob patterns up to a max depth
+func (l *Log) globPatterns(pattern string) ([]string, error) {
+	if l.config.DoubleWildcardMaxDepth == 0 {
+		return []string{pattern}, nil
+	}
+	var patterns []string
+	isAbs := filepath.IsAbs(pattern)
+	patternList := strings.Split(pattern, "/")
+	for i, dir := range patternList {
+		if len(patterns) > 0 {
+			if dir == "**" {
```

Comment:
Yes, we were discussing with @tsg and if we allow that the complexity will escalate quickly